.test-infra/dataproc/init-actions/beam.sh

```diff
@@ -44,8 +42,24 @@ function pull_images() {
   done
 }
 
+function pull_job_server() {
+  local job_server_location="$(/usr/share/google/get_metadata_value \
+    "attributes/${JOB_SERVER_IMAGE_METADATA_KEY}")"
+
+  if [[ -n "$job_server_location" ]] ; then
+    sudo -u yarn -i docker pull ${job_server_location}
+  fi
+
+}
+
 function main() {
+  local role="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
+
   pull_images
+
+  if [[ "${role}" == 'Master' ]] ; then
+    pull_job_server || err "Unable to start master"
```

Comment:
Unable to start master >> Unable to start Beam Job Server on Flink Master

Reply:
We're not starting the job server here (yet). It's done in `create_flink_cluster.sh` after running all init actions. It's only pulled here.

Either way, it's a good point, thanks! I changed to "Unable to pull Beam Job Server image on Flink Master"